ports/samd/mcu/samd51/mpconfigmcu.mk

```diff
@@ -1,7 +1,16 @@
 MICROPY_VFS_LFS2 ?= 1
+MICROPY_VFS_FAT ?= 1
 
 SRC_S += shared/runtime/gchelper_m3.s
 
+SRC_C += \
+	fatfs_port.c \
+	extmod/modonewire.c \
+	drivers/dht/dht.c \
```

Comment:
And very minor, this should now be in `DRIVERS_SRC_C` instead of `SRC_C`.

Reply:
I've noticed that already, as it prevented buidling.